src/Microsoft.Azure.WebJobs.Host/Scale/IScalerProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    /// <summary>
+    /// Interface for providing scalers.
+    /// </summary>
+    public interface IScalerProvider
```

Comment:
We might consider splitting this interface apart into two separate interfaces. I need to think about it more. The main reason would be single responsibility. In my prototype I shoved them together like this for simplicity, and also because the name IScaleMonitorProvider was already taken.

Reply:
what if we will use the names?
```
IScaleMonitorsProvider
ITargetScalersProvider
```

Reply:
We decided to IScaleMonitorProvider